app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -196,6 +196,62 @@
   border-bottom: 2px solid #aaa;
 }
 
+.panel-boundary-bottom {
+  padding-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+  border-bottom: 2px solid #aaa;
+}
+
+.panel-boundary-right {
+  border-right: 1px solid #ddd;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
 .book-title-table {
   max-width: 600px;
 }
+
+.panel-title {
+  margin-left: 10px;
+  font-size: 23px;
+  margin-top: 5px;
+  padding-bottom: 5px;
+}
+
+.book-search-panel-left {
+  width: 49%;
+  float: left;
+  padding-right: 10px;
+}
+
+.book-search-panel-right {
+  width: 49%;
```

Comment:
Properties should be ordered border-left, float, margin-bottom, margin-left, margin-top, padding-right, width